drivers/overlay/ov_network.go

```diff
@@ -298,6 +298,14 @@ func (n *network) initSubnetSandbox(s *subnet) error {
 	return nil
 }
 
+func (n *network) cleanupStaleSandboxes() {
+	for i := 1; i < n.initEpoch; i++ {
```

Comment:
Shouldn't be `<= n.initEpoch` given at creation time the counter is incremented before using it to name the sandbox ?